src/refract/JSONSchemaVisitor.cc

```diff
@@ -144,6 +151,15 @@ namespace refract
         pObj->push_back(m);
     }
 
+    void JSONSchemaVisitor::addDefinition(const std::string& key, IElement *val)
+    {
+        MemberElement *m = new MemberElement;
+
+        m->renderType(IElement::rCompact);
+        m->set(key, val);
+        pDefs->push_back(m);
+    }
```

Comment:
Next argument to do not define `ObjectElement::addMember()`
Nobody will remember special function to add member :)

```
pDefs->push_back(new MemberElement(key, val, IElement::rCompact));
```